pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-creation/metrics-collector/types.ts

```diff
@@ -0,0 +1,9 @@
+export enum CollectorKind {
```

Comment:
Should we specify these values in `app/constants/` ?

Reply:
I moved them under `app/enumerations` and removed the `types.ts` file.
Have also pushed a commit for this, so I'll resolve the convo